service/resource/cloudformationv1/adapter/outputs.go

```diff
@@ -0,0 +1,18 @@
+package adapter
+
+import (
+	"github.com/giantswarm/aws-operator/service/key"
+	"github.com/giantswarm/awstpr"
+)
+
+// template related to this adapter: service/templates/cloudformation/outputs.yaml
+
+type outputsAdapter struct {
+	ClusterVersion string
+}
+
+func (o *outputsAdapter) getOutputs(customObject awstpr.CustomObject, clients Clients) error {
+	o.ClusterVersion = key.VersionBundleVersion(customObject)
```

Comment:
I think this should be key.ClusterVersion. That gets the k8scloudconfig version from the custom object. If that changes we need to trigger an update to replace the nodes.

The version bundle version is used by the router to decide which version of the resources to execute. I don't think we need an output for that. As its part of the operator and not the cloud formation resource.

WDYT?

Reply:
Totally agree, I missed that, fixed.